cob_bringup/robots/cob3-2.xml

```diff
@@ -0,0 +1,177 @@
+<?xml version="1.0"?>
+<launch>
+
+	<!-- args -->
+	<arg name="robot" value="cob3-2"/>
+	<arg name="pc1" default="localhost"/>
+	<arg name="pc2" default="localhost"/>
+	<arg name="pc3" default="localhost"/>
+
+	<!-- upload robot description -->
+	<include file="$(find cob_hardware_config)/common/upload_robot.launch" >
+		<arg name="robot" value="$(arg robot)" />
+	</include>
+
+	<!-- upload default configuration parameters -->
+	<include file="$(find cob_default_robot_config)/cob3-2/upload_param_cob3-2.launch" />
+
+	<group>
+		<machine name="pc1" address="$(arg pc1)" env-loader="$(find cob_bringup)/env.sh" default="true" timeout="30"/>
+
+		<!-- pc monitor -->
+		<include file="$(find cob_bringup)/tools/pc_monitor.launch" >
+			<arg name="robot" value="$(arg robot)" />
+			<arg name="pc" value="$(arg pc1)" />
+		</include>
+
+		<!-- monitoring -->
+		<!--include file="$(find cob_bringup)/tools/wifi_monitor.launch" >
+			<arg name="robot" value="$(arg robot)" />
+		</include-->
```

Comment:
it is not working , cob3-2 has a different router, i want to keep it as comment, maybe this router is in future releases supported...